src/workers/cdc.ts

```diff
@@ -385,6 +386,9 @@ const onUserStateChange = async (
       await notifySubmissionGrantedAccess(logger, data.payload.after.userId);
     }
   }
+  if (data.payload.op === 'd') {
+    await notifyUserDeleted(logger, data.payload.before.userId, true);
```

Comment:
At the moment this will cause this message to be fired two times.
The current application version sends the request to the gateway which deletes the account on its behalf, then triggers a `user-deleted` message, API picks it up and delete, API sends another message.
Are there any side effects we have to be aware of?

Reply:
Hmm you are right the gateway retriggers now.
I tried to make it work with both systems, not quite sure how we can achieve that way to work.

Alternatively we can create a new topic?
Kratos could then subscribe to both versions?

Reply:
As long as there are no significant side effects, it's ok. Can you check for potential side effects, please?

Reply:
The only side effect I could think about is that it can cause the subscription to keep re-trying as it will already be deleted, but unsure how we can safely test it out.

I don't see any other side effects, but it's a hard one to try from the current local setup.
Might have to setup the new system for it.

Reply:
I don't see anything that should fail in case the user doesn't exist. It should go smoothly. Just in case, we can add a test case for this scenario

Reply:
Will add a test for it 🙏